packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -44,6 +44,7 @@ import { gotScraping } from 'got-scraping';
 import type { ProcessedRequest, Dictionary, Awaitable, BatchAddRequestsResult, SetStatusMessageOptions } from '@crawlee/types';
 import { chunk, sleep } from '@crawlee/utils';
 import ow, { ArgumentError } from 'ow';
+import { getDomain } from 'tldts';
```

Comment:
undeclared dependency

Reply:
Didn't get this... This library is already present in package.json.

Reply:
nope, it is not present in this package

https://github.com/apify/crawlee/blob/master/packages/basic-crawler/package.json#L47